### ISSUE TITLE
Redundancy: Find Dawn Fortress

### ISSUE THREAD

**Issue by askebv** [Heishe/idleraiders-second-run#491]
Yesterday I found the map to Dawn Fortress. Today I found the prophecy to find the map. Perhaps the prophecy droprate should be increased - and perhaps it should just auto-complete if you use the Map to Dawn Fortress.

**Comment by TravisGesslein** [comment]
Yep all those prophecies are supposed to auto complete if you've already fulfilled their requirements. Will look into it.
